src/sentry/api/endpoints/group_similar_issues.py

```diff
@@ -53,7 +53,7 @@ def get(self, request, group):
 
         # We need to preserve the ordering of the Redis results, as that
         # ordering is directly shown in the UI
-        for group_id, scores in list(zip(group_ids, group_scores)):
+        for group_id, scores in zip(group_ids, group_scores):
```

Comment:
This is not generally safe because the underlying collection may change within the iteration which previously did not cause an error, now it will.

Reply:
We can take a look at if the for loop is doing that though.

But that is a good point.